provider/lib/lib.go

```diff
@@ -13,6 +14,11 @@ var builtinConfig = Config{
 	Location: ".",
 }
 
+type Capability struct {
+	Capability      string
```

Comment:
Capability.Capability stutters I think, I think that we maybe want it to be the Name of the cap?